### ISSUE TITLE
Sticker minimum size

### ISSUE THREAD

**Issue by cainan** [wuapnjie/StickerView#85]
Is there a way to configure the sticker minimum size?

**Comment by ankurbhut** [comment]
Yes, But For this you need to add as a module and set size of sticker_transparent_background.xml file.

**Comment by MuhammedRefaat** [comment]
what do you mean by 'as a module"
